packages/server-wallet/CHANGELOG.md

```diff
@@ -3,6 +3,105 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [1.20.0](https://github.com/statechannels/statechannels/compare/@statechannels/server-wallet@1.14.1...@statechannels/server-wallet@1.20.0) (2021-02-05)
+
+
+### Bug Fixes
+
+* a finalized channel is incorrectly added back to the list after subscribers are notified ([296ad5a](https://github.com/statechannels/statechannels/commit/296ad5af9873cbf5eda7bdc44b8cab0ff7c54ce8))
```

Comment:
@kerzhner This wording is a bit confusing as a commit message header
```suggestion
* do not add finalized channel if ...([296ad5a](https://github.com/statechannels/statechannels/commit/296ad5af9873cbf5eda7bdc44b8cab0ff7c54ce8))
```

"a finalized channel is incorrectly added back to the list after subscribers are notified" seems like it belongs in the body of the msg?